var/spack/repos/builtin/packages/py-nvidia-dali/package.py

```diff
@@ -23,10 +23,22 @@ class PyNvidiaDali(PythonPackage):
     system = platform.system().lower()
     arch = platform.machine()
     if "linux" in system and arch == "x86_64":
+        version(
+            "1.41.0-cuda120",
+            sha256="240b4135e7c71c5f669d2f2970fa350f7ad1a0a4aab588a3ced578f9b6d7abd9",
+            preferred=True,
```

Comment:
I noticed you didn't add `preferred=True` to the aarch versions. Since these are anyway the most recent version, is there any reason to explicitly mark them as preferred? In any case, we should have either both x86_64 and aarch preferred, or both using default preference.

Reply:
Indeed, this is no longer necessary. Thank you.